src/main/scala/cromwell/Main.scala

```diff
@@ -18,11 +20,23 @@ object Actions extends Enumeration {
 
 object Main extends App {
 
+  val props = System.getProperties
+  val loggerProperty = "CROMWELL_LOGGER"
+  Option(props.getProperty(loggerProperty)) match {
```

Comment:
I think this would read cleaner w/ a standard `getOrElse` and the else calls a func which does the stuff in the first `None` case.